website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -178,6 +176,8 @@ The following arguments are supported:
 
 -> **NOTE:** One of either `source_image_id` or `source_image_reference` must be set.
 
+~> **NOTE:** If `source_image_id` is set to a shared image with `os_state` of `Specialized`, or a shared image version created by a specialized shared image, `admin_username`, `admin_password`, `custom_data`, `admin_ssh_key` and `secret` must not be specified.
```

Comment:
we should add a disclaimer about Specialized Images here, since reusing the machine UUID can cause data loss in a clustered scenario:

```suggestion
~> **NOTE:** If `source_image_id` is set to a shared image with `os_state` of `Specialized`, or a shared image version created by a specialized shared image, `admin_username`, `admin_password`, `custom_data`, `admin_ssh_key` and `secret` must not be specified.

!> **Note:** Specialized Images reuse the UUID of the Virtual Machine which can have unexpected consequences, including data corruption (for example, when running in a cluster and two nodes share the same ID) - as such we recommend using a Generalized image where possible.
```

this'll also want to go up the top in the "disclaimers" section

Reply:
Done and disclaimers added